app/src/main/java/com/example/steamapp/presentation/ui/ownedgames/GameAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.steamapp.presentation.ui.ownedgames
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.steamapp.databinding.GameLstElemBinding
+import com.example.steamapp.domain.model.Game
+
+class GameAdapter(
+    context: Context
+) : ListAdapter<Game, RecyclerView.ViewHolder>(DIFF_UTIL) {
```

Comment:
в адаптере используется 1 тип вью холдера, не зря тут параметризация есть, можно так, чтобы проверок лишних не писать
```suggestion
) : ListAdapter<Game, GameViewHolder>(DIFF_UTIL) {
```